infrastructure/terraform/roles.tf

```diff
@@ -45,3 +45,181 @@ resource "kubernetes_role_binding" "job_robot" {
     api_group = "rbac.authorization.k8s.io"
   }
 }
+
+# Service account for the kubernetes cluster autoscaler
+resource "kubernetes_service_account" "cluster_autoscaler" {
```

Comment:
> IMO everything from here down should be put into a new file, since it's pretty separate from what's going on here. Maybe `cluster-autoscaler-roles.tf` or something.

https://github.com/ResonantGeoData/RD-OASIS/pull/43/commits/ac4766110012ffde1a67ec95ce9041a5e62cedc6

> We also might want to consider splitting some of this up into modules, since the cluster autoscaling stuff is pretty isolated. We could do that on another pass though.

:+1: